Module not found: Error: Can't resolve './dist/svg/ds6/icons.svg.css'

it looks like the file extension could be generated wrong (   “@ebay/skin”: “12.3.1",)
ERR! Module not found: Error: Can’t resolve ‘./dist/svg/ds6/icons.svg.css’

<img width="928" alt="Screen Shot 2021-04-05 at 11 11 44 AM" src="https://user-images.githubusercontent.com/8145951/113609786-1e27f180-9601-11eb-9f46-ddda876d0c7a.png">

--- a/scripts/generate-imports/index.js
+++ b/scripts/generate-imports/index.js
@@ -39,7 +39,7 @@ function getCSSRequireSyntax(filepath, ext) {
 
 function getJSRequireSyntax(filepath, ext) {
     let fullFilePath = `${filepath}.${ext}`;
-    if (filepath.indexOf('.css') === filepath.length - 4) {
+    if (filepath.indexOf('.css') === filepath.length - 4 || filepath.includes('svg')) {
         fullFilePath = filepath;
     }
     return `require('./${fullFilePath}');\n`;
@@ -115,7 +115,6 @@ async function generateTopLevelFiles() {
     const indexFiles = browserRemap.filter(
         (item) => config.skipIndex.indexOf(item.filename) === -1
     );
-
     const contentJS = indexFiles
         .map((item) => {
             return `require('./${item.filename}.js');\n`;
@@ -131,7 +130,6 @@ async function generateTopLevelFiles() {
             return `@import "./${item.filename}.css";\n`;
         })
         .join('');
-
     await fs.promises.writeFile(
         path.join(currentDir, 'browser.json'),
         prettier.format(JSON.stringify(browser), { parser: 'json' })

